.github/workflows/fossa.yml

```diff
@@ -0,0 +1,21 @@
+name: FOSSA
+
+on:
```

Comment:
Is there a reason to not also run this on PRs? It'd be ideal to catch these issues before the PR gets merged.

Reply:
I agree, but the scan requires an API key, which is stored in a secret, which is not accessible to forks, so this was the next best thing.

Reply:
Ah, that makes sense.